Purchasing.Jobs.Common/packages.config

```diff
@@ -1,11 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="CommonServiceLocator" version="1.3" targetFramework="net45" />
-  <package id="elmah.io.client" version="2.0.22" targetFramework="net451" />
+  <package id="Elasticsearch.Net" version="7.8.1" targetFramework="net462" />
+  <package id="elmah.io.client" version="2.0.29" targetFramework="net462" />
+  <package id="Microsoft.CSharp" version="4.6.0" targetFramework="net462" />
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.1.0" targetFramework="net451" />
   <package id="Ninject" version="3.2.2.0" targetFramework="net45" />
-  <package id="Serilog" version="1.5.7" targetFramework="net451" />
-  <package id="Serilog.Sinks.ElmahIO" version="1.5.8" targetFramework="net451" />
+  <package id="Serilog" version="2.8.0" targetFramework="net462" />
+  <package id="Serilog.Formatting.Compact" version="1.0.0" targetFramework="net462" />
+  <package id="Serilog.Formatting.Elasticsearch" version="8.4.1" targetFramework="net462" />
+  <package id="Serilog.Sinks.Elasticsearch" version="8.4.1" targetFramework="net462" />
+  <package id="Serilog.Sinks.ElmahIO" version="2.0.13" targetFramework="net462" />
```

Comment:
wow are we still writing anything to elmah?

Reply:
Other than an import, I don't see any actual code that references it.

Reply:
Cool well leave it in but i'll make an issue.  I'm sure not checking elmah anymore, or even know where i would check

Reply:
@srkirkland I'm glad you mentioned the history page. Got a TypeLoadException because NEST was outdated.